spec/requests/users_spec.rb

```diff
@@ -25,4 +25,21 @@
       end
     end
   end
+  path '/{locale}/check_username?name={username}' do
```

Comment:
Why we need this path? Why not `path '/{locale}/check_username?name={username}' do`

Reply:
We have a parameter in the URL username.

Reply:
It's because in the swagger to specify the parameters, we must to add them to url, in postman we can use the short version of path